imports/plugins/core/checkout/client/components/cartIcon.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component, PropTypes } from "react";
+import { Reaction } from "/client/api";
+
+class CartIcon extends Component {
+  static propTypes = {
+    cart: PropTypes.object
+  }
+
+  handleClick = (event) => {
+    event.preventDefault();
+    return $("#cart-drawer-container").fadeOut(300, function () {
```

Comment:
Findings:

* This is not the only component in the cart icon/panel/drawer code with jquery code. The fade in part of the code is handled in the parent template, cartDrawer, while the fadeout is handled in separate components: CartIcon, CartPanel, and CartDrawer.
* I’m not sure how to remove the query code in the cart icon alone and not affect the others.

Solutions:

* add // TODO: Remove Jquery blocks 
* Remove the fade in/fade out animations temporarily, and figure out another way to do them.